CMakePresets.json

```diff
@@ -0,0 +1,151 @@
+{
+    "configurePresets": [
+        {
+            "name": "base",
+            "hidden": true,
+            "cacheVariables": {
+                "CMAKE_CXX_COMPILER_LAUNCHER": {
+                    "value": "ccache",
+                    "type": "STRING"
+                },
+                "CMAKE_CUDA_COMPILER_LAUNCHER": {
+                    "value": "ccache",
+                    "type": "STRING"
+                },
+                "FLASHINFER_DECODE": {
+                    "value": "ON",
+                    "type": "STRING"
+                },
+                "FLASHINFER_PREFILL": {
+                    "value": "ON",
+                    "type": "STRING"
+                },
+                "FLASHINFER_PAGE": {
+                    "value": "ON",
+                    "type": "STRING"
+                },
+                "FLASHINFER_CASCADE": {
+                    "value": "ON",
+                    "type": "STRING"
+                },
+                "FLASHINFER_SAMPLING": {
+                    "value": "ON",
+                    "type": "STRING"
+                },
+                "FLASHINFER_NORM": {
+                    "value": "ON",
+                    "type": "STRING"
+                },
+                "FLASHINFER_CUDA_ARCHITECTURES": {
+                    "value": "90a",
```

Comment:
Hopper only is a good default?

Reply:
Not necessarily. I would default to whichever compute capability can be detected on the host. In theory, the string `native` should be recognized by cmake, but I need to track all the usages of this variable.